src/runtime/internal/routes/openapi.ts

```diff
@@ -41,21 +41,24 @@ export default eventHandler((event) => {
       },
     ],
     paths,
-    components,
+    components: globals.components,
   };
 });
 
-function getOpenApiMeta(): DeepPartial<OpenAPI3> {
+type OpenAPIGlobals = Pick<OpenAPI3, "components">;
```

Comment:
I think we should move this type from here to `src/types/openapi.ts`

Reply:
@pi0 one more thing here

Reply:
Perhaps latest when we wanted to add more. adds additional imports to handler.ts and intenral/openapi.ts